com.valvesoftware.Steam.CompatibilityTool.Proton.metainfo.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>com.valvesoftware.Steam.CompatibilityTool.Proton</id>
+  <extends>com.valvesoftware.Steam</extends>
+  <name>Proton (community build)</name>
+  <summary>Compatibility tool for Steam Play based on Wine and additional components</summary>
+  <summary xml:lang="pt_BR">Ferramenta de compatibilidade para Steam Play baseada no Wine e componentes adicionais</summary>
+  <summary xml:lang="es">Herramienta de compatibilidad para Steam Play basada en Wine y componentes adicionales</summary>
```

Comment:
Update metainfo translation to pt_BR and es for Proton (Community build)
```suggestion
  <name>Proton (Community build)</name>
  <name xml:lang="pt_BR">Proton (Construção da comunidade)</name>
  <name xml:lang="es">Proton (Construcción comunitaria)</name>
  <summary>
  <p>Compatibility tool for Steam Play based on Wine and additional components</p>
  <p>This is a community package of Steam Play plaform not officially supported by Valve. Report bugs through linked issue tracker.</p> 
  </summary>
  <summary xml:lang="pt_BR">
  <p>Ferramenta de compatibilidade para Steam Play baseada no Wine e componentes adicionais</p>
  <p>Este é um pacote comunitário da plataforma Steam Play não oficialmente suportado pela Valve. Relatar bugs por meio do rastreador de problemas vinculado.</p>
  </summary>
  <summary xml:lang="es">
  <p>Herramienta de compatibilidad para Steam Play basada en Wine y componentes adicionales</p>
  <p>Este es un paquete comunitario de plataforma Steam Play que Valve no admite oficialmente. Informar errores a través del rastreador de problemas vinculado.</p>
  </summary>
```

Reply:
`summary` tag can't include markup and it shouldn't be this long.